### PR TITLE
update UserGuide to include module add task

### LINKED ISSUE #11: Update User Guide for adding a task to a module

### PR DESCRIPTION
**Pull request by charliemoweng**
Fix #11

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+TAB3 is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still 
+having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
```
Comment (Ruofan1023) [inline docs/UserGuide.md:7]: change AB3 tp TAB3?

Comment (charliemoweng) [reply docs/UserGuide.md:7]: ok on it

**Comment by Ruofan1023** [comment]
looks good to merge

**Comment by tanjoewel** [comment]
LGTM!

**Pull request merged**